### ISSUE TITLE
Ipad app not working

### ISSUE THREAD

**Issue by gasserwshalaby** [FlutterFlow/flutterflow-issues#4945]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

the flutterflow app just gives a black screen and doesn’t even go through to the dashboard of my apps.

### Expected Behavior

to work normally like the webapp or windows/mac apps

### Steps to Reproduce

1. download the ipad flutterflow app
2. start it
3. you’ll find a black screen

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

000000000

### Visual documentation

![IMG_0148](https://github.com/user-attachments/assets/d70396eb-fba2-4f39-bceb-6f45e4b29574)
screenshot from the app

### Environment

```markdown
- FlutterFlow version: 5.0.13
- Platform: ipad
- Browser name and version: ipad app
- Operating system and version affected: ipad os v 17.5.1
```

**Comment by Alezanello** [comment]
Hello!!

I’m sorry to hear about this experience. I can confirm that this issue exists, and I’ll be forwarding it to the engineering team to work on a fix. Thank you for bringing it to our attention!

**Comment by stepanic** [comment]
I am facing this issue also, temp fix I just a revert to 5.0.10.

I've tried to install it on iPad 10th and Mac mini M4 on both device blank screen appears.

@Alezanello please pink an engineering team to release previous build again, they probably did already but I guess that Apple takes some time again to review it.

If you can provide IPA file somewhere to download then we could be able to install it with 
https://sidestore.io/

I found the previous binary here
https://ipafilezone.com/app/usa/6457995947
https://ipafilezone.com/download/fd96c1e39508c50304cc3a7dedb94b7f1f5ae18d42f5522904e6e01268515a07
and I installed it on iPad and it is possible to run but I don't want to insert my FlutterFlow login credentials there because I don't know does that binary contains malware which can stole my FF credentials but FlutterFlow still does not have an 2FA authentication.

**Comment by stepanic** [comment]
> I am facing this issue also, temp fix I just a revert to 5.0.10.
> 
> I've tried to install it on iPad 10th and Mac mini M4 on both device blank screen appears.
> 
> @Alezanello please pink an engineering team to release previous build again, they probably did already but I guess that Apple takes some time again to review it.
> 
> If you can provide IPA file somewhere to download then we could be able to install it with https://sidestore.io/
> 
> I found the previous binary here https://ipafilezone.com/app/usa/6457995947 https://ipafilezone.com/download/fd96c1e39508c50304cc3a7dedb94b7f1f5ae18d42f5522904e6e01268515a07 and I installed it on iPad and it is possible to run but I don't want to insert my FlutterFlow login credentials there because I don't know does that binary contains malware which can stole my FF credentials but FlutterFlow still does not have an 2FA authentication.

Please check the screen recording where it is visible that both version of FlutterFlow iOS app are installed at the same time on the same device.

![GMP_RmlsZXNHTTAw](https://github.com/user-attachments/assets/b12b21ec-3c67-4cfc-a15a-51c5d84a6ffc)

**Comment by sbis04** [comment]
This should be fixed in the latest version!
